site/en/r2/tutorials/distribute/multi_worker.ipynb

```diff
@@ -346,37 +351,39 @@
       "outputs": []
     },
     {
+      "cell_type": "markdown",
       "metadata": {
         "id": "XVk4ftYx6JAO",
         "colab_type": "text"
       },
-      "cell_type": "markdown",
       "source": [
-        "## Performance\n",
+        "## Optimize training performance\n",
         "\n",
-        "You now have a model and a multi-worker capable Estimator powered by `tf.distribute.Strategy`.  You can try the following techniques to tweak performance of multi-worker training.\n",
+        "You now have a model and a multi-worker capable Estimator powered by `tf.distribute.Strategy`.  You can try the following techniques to optimize performance of multi-worker training:\n",
         "\n",
-        "*   `MultiWorkerMirroredStrategy` provides multiple [collective communication implementations](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/distribute/cross_device_ops.py).  `RING` implements ring-based collectives using gRPC as the cross-host communication layer.  `NCCL` uses [Nvidia's NCCL](https://developer.nvidia.com/nccl) to implement collectives.  `AUTO` defers the choice to the runtime.  The best choice of collective implementation depends upon the number and kind of GPUs, and the network interconnect in the cluster.  To override the automatic choice, specify a valid value to the `communication` parameter of `MultiWorkerMirroredStrategy`'s constructor, e.g. `communication=tf.distribute.experimental.CollectiveCommunication.NCCL`.\n",
-        "*   The batch size specified here is per-GPU.  In general, the largest batch size that fits the GPU memory is advisable.\n",
-        "*    Cast the variables to `tf.float` if possible.  The official ResNet model includes [an example](https://github.com/tensorflow/models/blob/8367cf6dabe11adf7628541706b660821f397dce/official/resnet/resnet_model.py#L466) of how this can be done.\n",
-        "\n",
-        "\n",
-        "\n"
+        "*   **Increase the batch size:** The batch size specified here is per-GPU.  In general, the largest batch size that fits the GPU memory is advisable.\n",
```

Comment:
Remove the bold styles here in the list (italics/emphasis is fine). It visually competes with the headings on the site design

Reply:
done